solarforecastarbiter/io/reference_observations/default_forecasts.py

```diff
@@ -112,5 +112,57 @@ def is_in_nwp_domain(site):
 ]
 
 
+TEMPLATE_DETERMINISTIC_PERSISTENCE_FORECASTS = [
```

Comment:
How about a day ahead persistence with hour-ending intervals?

Reply:
I guess I made the run length 1h, is that what you meant or 1day?

Reply:
I meant 1 day, as in line 30 of the workshop repo example: https://nbviewer.jupyter.org/github/SolarArbiter/workshop/blob/master/reference_forecasts.ipynb#persistence-forecasts